NYPLCardCreator/BusinessLogic/FlowCoordinator+Juvenile.swift

```diff
@@ -202,7 +134,7 @@ public class JuvenileFlowCoordinator {
     // contain a valid url. This nil-coalescing check is just for future-proofing.
     let urlStr = req.url?.absoluteString ?? "missing URL from eligibility request"
 
-    let task = urlSession.dataTask(with: req) { data, response, error in
+    let task = self.urlSession.dataTask(with: req) { data, response, error in
```

Comment:
nit: just wondering if this `self.` was necessary... most swift code I've seen omits it if possible